.github/workflows/containers.yml

```diff
@@ -8,18 +8,15 @@ on:
     branches: [master]
 
 jobs:
-  container-dev:
-    name: "Dev container"
+  build-fcos:
+    name: "Build in FCOS buildroot"
     runs-on: ubuntu-latest
+    container: quay.io/coreos-assembler/fcos-buildroot:testing-devel
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
-      - name: Assemble dev-container
-        run: docker build -f contrib/Dockerfile.dev -t coreos/zincati:dev .
-      - name: Run dev-container
-        run: docker run --rm -v "$(pwd):/source" -v "/tmp/assembler:/assembler" coreos/zincati:dev
-      - name: Archive artifacts (overrides/rootfs)
-        uses: actions/upload-artifact@v2
-        with:
-          name: zincati-overrides-rootfs
-          path: /tmp/assembler/overrides/rootfs/
```

Comment:
I actually have a question about this part in the original code though, what is the purpose of this?

Reply:
It does persist built artifacts into an archive. You can see it under "Artifacts" on other PRs, like here: https://github.com/coreos/zincati/pull/524/checks.

I never really used so I'm not too worried to see it go, but longer term it would be nice to have auto-built binaries ready to download (in the form of an RPM, or ostree content, or a tarball, or whatever).

Reply:
ah, I see. Thanks!